plugins/ps/pre-deploy

```diff
@@ -5,4 +5,5 @@ source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
 APP="$1"
 
+get_procfile $APP
```

Comment:
Ah I missed that. Personally, I'd like to get away from cluttering the `/home/dokku/APP` directory. However, I believe that concern is being addressed in another issue (#1558)